modules/BuildTools.Tasks/BuildTools.Tasks.csproj

```diff
@@ -9,16 +9,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\..\shared\Microsoft.Extensions.CommandLineUtils.Sources\**\*.cs" />
```

Comment:
Why does the feed location make a difference? I'm not trying to disavow checking in sources, I"m just trying to figure out how any of this is meant to work.